contentful/client.py

```diff
@@ -219,10 +219,14 @@ def entry(self, entry_id, query=None):
 
         try:
             query.update({'sys.id': entry_id})
-            return self._get(
+            response = self._get(
                 self.environment_url('/entries'),
                 query
-            )[0]
+            )
+
+            if self.raw_mode:
+                return response
+            return response[0]
```

Comment:
Trying to fetch a single entry with `raw_mode=True` throws this error for me:

`TypeError: 'Response' object is not subscriptable`

Reply:
@gilesbrunning I am taking a look at this issue now. Will keep you posted on my progress

Cheers

Reply:
@gilesbrunning I could not reproduce the error. How are you retrieving a single entry? Is there a specific entry you are facing error with? Which version of contentful.py library are you using? If it is not latest can you please update it to latest version and let me know if the problem still persists.

Cheers

Reply:
Hi @rubydog , I was using 1.12.3 but when I upgraded to 1.13.1 the problem is gone so all good. Thanks for checking.